### ISSUE TITLE
Pick List, Pick List Group and Pick List Item - Ensure that the reading order of content is logical - (2036570122)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#5597]
### Module: 
 24 Pick List, Pick List Group and Pick List Item 

### Violation: 
 Ensure that the reading order of content is logical 

![image](https://user-images.githubusercontent.com/5023024/202743150-f19fc5d2-16b7-4719-b4d8-2a9cfef9e067.png)

### WCAG Reference: 
- [1.3.2 Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) 

### Instance ID: 
 2036570122 

### Severity: 
 8 

### Description: 
 [Issue]
The following reading order is illogical:
1. bulleted icon that shows selection of list item
2. button Enable River layer

[User Impact]
Screen reader users will be unable to understand this content.

[Code Reference] 
 ```
<span class="icon icon-dot"></span>
(...)
<button aria-busy="false" aria-disabled="false" aria-label="Enable River layer" class="button"><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="layer" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container">Enable River layer</div></button>
 ``` 

### Note: 
 [Suggestion]
Ensure that the reading order of content is logical. The best way to set the reading order is to reorder the elements in the DOM.
Correct reading order:
1. bulleted icon that shows selection of list item
2. River
3. A large natural stream of water flowing in a channel to the sea, a lake, or another such stream.
4. button Enable River laye

[Compliant Code Example] 
 ```
<span class="icon icon-dot"></span>
<label aria-label="River" class="label" tabindex="0"><div aria-checked="false" class="text-container" role="menuitemcheckbox"><span class="title">River</span>
<span class="description">A large natural stream of water flowing in a channel to the sea, a lake, or another such stream.</span></div></label>
(...)
<button aria-busy="false" aria-disabled="false" aria-label="Enable River layer" class="button"><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="layer" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container">Enable River layer</div></button>
 ```
 

### Media Type: 
 Page Structure 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This should be addressed in the updated `list` component.

**Comment by driskull** [comment]
We can ignore this one for now and direct users to use the list component

**Comment by geospatialem** [comment]
Adding to our post 1.0 priorities project to determine if we're hitting the mark with the updated `list` component.

**Comment by geospatialem** [comment]
Closing as the `pick-list` components are deprecated and functionality will not be supported at this time.
